groovy/prePostInit/Thermodynamics.groovy

```diff
@@ -748,17 +748,17 @@ for (WorkingFluid in WorkingFluids) {
     }
         
     recipemap('cooling_tower').recipeBuilder()
-            .fluidInputs(liquid(WorkingFluid.leftover_fluid) * (WorkingFluid.amount_to_use * WorkingFluid.conversion_factor) * 64)
+            .fluidInputs(liquid(WorkingFluid.leftover_fluid) * (WorkingFluid.amount_to_use * WorkingFluid.conversion_factor))
             .fluidInputs(liquid('water') * 1000)
-            .fluidOutputs(liquid(WorkingFluid.normal_fluid) * WorkingFluid.amount_to_use * 64)
+            .fluidOutputs(liquid(WorkingFluid.normal_fluid) * WorkingFluid.amount_to_use)
```

Comment:
The correct expression would have been to multiply the liquid by (WorkingFluid.amount_to_use * 64).